test/e2e/delete_test.go

```diff
@@ -85,6 +89,18 @@ func (tc *testContext) checkDirsDoNotExist(address string) (bool, error) {
 	return !strings.Contains(out, "exists"), nil
 }
 
+// checkNetworksRemoved returns true if the HNS networks created by hybrid-overlay do not exist on the Windows
+// instance with the given address
+func (tc *testContext) checkNetworksRemoved(address string) (bool, error) {
+	command := fmt.Sprintf("powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"Get-HnsNetwork\"")
```

Comment:
I am noticing that `powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command` is being repeated in the test code and warrants a constant.